examples/forum/schema.sql

```diff
@@ -2,111 +2,106 @@
-comment on column post.author_id is 'The id of the author user.';
-comment on column post.topic is 'The topic this has been posted in.';
-comment on column post.body is 'The main body text of our post.';
-comment on column post.created_at is 'The time this post was created.';
-comment on column post.updated_at is 'The latest time this post was updated.';
+comment on table forum_example.person_account is 'Private information about a person’s account.';
+comment on column forum_example.person_account.person_id is 'The id of the person associated with this account.';
+comment on column forum_example.person_account.email is 'The email address of the person.';
+comment on column forum_example.person_account.pass_hash is 'An opaque hash of the person’s password.';
 
--------------------------------------------------------------------------------
--- Private Tables
+create type forum_example.post_topic as enum (
+  'discussion',
+  'inspiration',
+  'help'
+);
 
-create table forum_example_utils.person_account (
-  person_id        int not null primary key,
-  email            varchar not null unique check (email ~* '^.+@.+\..+$'),
-  pass_hash        char(60) not null
+create table forum_example.post (
+  id               uuid not null primary key default uuid_generate_v1mc(),
+  author_id        uuid not null references forum_example.person(id),
+  headline         text not null check (char_length(headline) < 500),
+  topic            forum_example.post_topic,
+  body             text,
+  created_at       timestamp,
+  updated_at       timestamp
 );
 
-comment on table person_account is 'Private information about a person’s account.';
-comment on column person_account.person_id is 'The id of the person associated with this account.';
-comment on column person_account.email is 'The email address of the person.';
-comment on column person_account.pass_hash is 'An opaque hash of the person’s password.';
+comment on table forum_example.post is 'A forum post written by a user.';
+comment on column forum_example.post.id is 'The primary key for the post.';
+comment on column forum_example.post.headline is 'The title written by the user.';
+comment on column forum_example.post.author_id is 'The id of the author user.';
+comment on column forum_example.post.topic is 'The topic this has been posted in.';
+comment on column forum_example.post.body is 'The main body text of our post.';
+comment on column forum_example.post.created_at is 'The time this post was created.';
+comment on column forum_example.post.updated_at is 'The latest time this post was updated.';
 
 -------------------------------------------------------------------------------
 -- Query Procedures
 
 -- Computes the full name for a person using the person’s `given_name` and a
 -- `family_name`.
-create function person_full_name(person) returns varchar as $$
-  select $1.given_name || ' ' || $1.family_name
+create function forum_example.person_full_name(person) returns text as $$
```

Comment:
`ERROR:  type person does not exist`
Need to set the schema in a few places in this file for `person` and `post`. e.g. `forum_example.person`.